### PR TITLE
Added new email templates and sending email address

### PR DESCRIPTION
**Pull request by MichaelChung123**
# Description

This PR is made to remove the BC Logo from all emails sent from CHEFS and also change the sending email to DoNotReplyCommonServices@gov.bc.ca. This is done to fix the issues with certain Email Clients from not showing the image at all and the email address was changed to be more suitable for an email coming from the CSS Team.

Original Jira Ticket:
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2082?

## Types of changes

I introduced a new Template for the emails being sent out, as well as removing the BC Logo

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

## Further comments

N/A

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-178

File: app/src/forms/email/assets/confirmation-number-email.html [hunk 1]
```diff
@@ -1,118 +1,259 @@
...
+                                      </td>
+                                    </tr>
+
+                                </table>
+                            </td>
+                          </tr>
+
+
+
+
```
Comment (jujaga) [inline app/src/forms/email/assets/confirmation-number-email.html:180]: Large number of unnecessary whitespace lines. Please make sure to run the VSCode formatter in the right click menu to make sure things get compacted and indented correctly.

File: app/src/forms/email/assets/confirmation-number-email.html [hunk 2]
```diff
@@ -1,118 +1,259 @@
...
+                    </div>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+  </div>
+  </td>
+  </tr>
+  </tbody>
+  </table>
```
Comment (jujaga) [inline app/src/forms/email/assets/confirmation-number-email.html:192]: Strange lack of indentation here.

File: app/src/forms/email/assets/status-assignment-email.html [hunk 3]
```diff
@@ -1,111 +1,259 @@
...
+                                          font-family: Oxygen, Helvetica neue, sans-serif;
+                                          font-size: 24px;
+                                          font-weight: 700;
+                                          line-height: 37px;
+                                          text-decoration: none;
+                                          display: block;
+                                        ">
+                                            {{ title }}
+                                          </div>
+                                        </a>
```
Comment (jujaga) [inline app/src/forms/email/assets/status-assignment-email.html:59]: This anchor attribute should not href to subscribe.developer.gov.bc.ca. Since we don't have a clear concept between user and staff for CHEFS yet, I'm going to suggest we remove the hyperlink from this entirely; instead just convert this to a div containing the title and it should be sufficient.

File: app/src/forms/email/assets/status-assignment-email.html [hunk 4]
```diff
@@ -1,111 +1,259 @@
...
-    <tr>
-      <td width="59" style="width:44.25pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="213" style="width:159.75pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="18" style="width:13.5pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="241" style="width:180.75pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="49" style="width:36.75pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="69" style="width:51.75pt; padding:0cm 0cm 0cm 0cm"></td>
-    </tr>
+
+
```
Comment (jujaga) [inline app/src/forms/email/assets/status-assignment-email.html:180]: Large number of unnecessary whitespace lines. Please make sure to run the VSCode formatter in the right click menu to make sure things get compacted and indented correctly.

File: app/src/forms/email/assets/status-assignment-email.html [hunk 5]
```diff
@@ -1,111 +1,259 @@
...
+                    </div>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+  </div>
+  </td>
+  </tr>
+  </tbody>
+  </table>
```
Comment (jujaga) [inline app/src/forms/email/assets/status-assignment-email.html:192]: Strange lack of indentation here.

**Review by jujaga — COMMENTED** [review]
Functionally working as intended - template looks fantastic! Please take a look at the following comments for general cleanup.

File: app/src/forms/email/assets/confirmation-number-email.html [hunk 6]
```diff
@@ -1,118 +1,259 @@
...
+                        </tbody>
+                      </table>
+                    </div>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <!-- Progress Modal Start -->
+            <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation"
+              style="background: #f7f7f7; background-color: #f7f7f7; width: 100%">
```
Comment (TimCsaky) [inline app/src/forms/email/assets/confirmation-number-email.html:77]: Minor aesthetic detail:  you might want a `max-width: 600px` to make the grey box more centered

Comment (TimCsaky) [reply app/src/forms/email/assets/confirmation-number-email.html:77]: oops nevermind.. i see you have it a few lines down. hmm

Comment (TimCsaky) [reply app/src/forms/email/assets/confirmation-number-email.html:77]: i guess we could have more padding on the 'View Submission' button.
I havent tried others but this may just happen in outlook desktop. very tricky to get fixed everywhere i believe.

Comment (MichaelChung123) [reply app/src/forms/email/assets/confirmation-number-email.html]: Made styling changes to fit the Outlook Desktop format.

**Review by jujaga — APPROVED** [review]

**Review by loneil — APPROVED** [review]
Looking good, I have a change in my PR to one of these templates so will handle the merge confict and re-add the new link in that PR .
